paddle/framework/eigen_test.cc

```diff
@@ -108,5 +108,25 @@ TEST(Eigen, Matrix) {
   }
 }
 
+TEST(Eigen, MatrixReshape) {
+  Tensor t;
+  float* p =
+      t.mutable_data<float>(make_ddim({2, 3, 6, 4}), platform::CPUPlace());
```

Comment:
`make_ddim` is not needed, just `t.mutable_data<float>({2, 3, 6, 4})` is cool.

Reply:
thanks!